conans/client/command.py

```diff
@@ -283,7 +284,8 @@ def test(self, *args):
         self._warn_python2()
         return self._conan.test(args.path, args.reference, args.profile, args.settings,
                                 args.options, args.env, args.remote, args.update,
-                                build_modes=args.build, test_build_folder=args.test_build_folder)
+                                build_modes=args.build, test_build_folder=args.test_build_folder,
+                                lockfile=args.lockfile)
```

Comment:
The `conan test` is using a new `install folder` argument and I don't know what is that for.

Reply:
Removing it too, not necessary